### DISCUSSION TITLE
Talk about GyroShell here.

### DISCUSSION THREAD

**Discussion by Pdawg-bytes** [Pdawg-bytes/GyroShell#8] [General]
Well, let's start the discussions!

**Comment by errortek** [comment]
@Pdawg-bytes I'm trying to compile GyroShell, and get a compile issue with the LocalSettings lines, but VS directs me to the System.Global.Debugger.Break() command, and thinks that causes the error, so I don't know exactly why it crashed when I built it.
Could you help please?

**Reply by Pdawg-bytes** [reply]
Hello! Make sure you're building the app with the **Packaged** setting. If not, you will encounter this error.

**Reply by errortek** [reply]
> Hello! Make sure you're building the app with the **Packaged** setting. If not, you will encounter this error.

alright, only issue I'm having is that building with Packaged is broken for me, it says 
![image](https://github.com/Pdawg-bytes/GyroShell/assets/100033309/e5a301fe-92cd-4756-a6cb-1eca9a56c1c4)

**Reply by errortek** [reply]
that's why I tried to compile it unpackaged

**Reply by errortek** [reply]
still doesn't work for Release/Packaged
maybe I don't have the correct files or smth
same error as before
![image](https://github.com/Pdawg-bytes/GyroShell/assets/100033309/797140de-fe10-4ece-bb6f-e79b55721421)

**Reply by errortek** [reply]
do you know a fix for that dialog @Pdawg-bytes ?

**Reply by Pdawg-bytes** [reply]
Install this extension:

https://marketplace.visualstudio.com/items?itemName=ProjectReunion.MicrosoftSingleProjectMSIXPackagingToolsDev17

**Reply by errortek** [reply]
> Install this extension:
> 
> https://marketplace.visualstudio.com/items?itemName=ProjectReunion.MicrosoftSingleProjectMSIXPackagingToolsDev17

ok thank you! :D

**Comment by errortek** [comment]
unfortunately, now, when I try to launch GyroShell it just crashes...
VS gives me this message: 

> The program '[24684] GyroShell.exe' has exited with code 4294967295 (0xffffffff).

**Reply by errortek** [reply]
sorry for constantly pinging you @Pdawg-bytes , I just need help with all this! :(

**Reply by errortek** [reply]
especially this crash 💀

**Reply by Pdawg-bytes** [reply]
Is there any other information? Can you send a screenshot of your whole screen right when it crashes?

**Reply by errortek** [reply]
> Is there any other information? Can you send a screenshot of your whole screen right when it crashes?

alright
so the only thing I did was compile GyroShell successfully and it just refuses to launch
video of what happens is attached

https://github.com/Pdawg-bytes/GyroShell/assets/100033309/1a2ee4a7-736b-4dad-9227-56aabd406a15

**Reply by Pdawg-bytes** [reply]
There's no information that can help me figure out what's wrong here, so please wait for further notice.

**Reply by errortek** [reply]
> There's no information that can help me figure out what's wrong here, so please wait for further notice.

alright, I can wait :D
It's not urgent! :D
